src/modules/pager/pager.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+var moduleName = 'c2fo.ui.modules.pager';
```

Comment:
It'd be nice to see what's exported further up the file like decided for the facade, this makes a nice table of contents for people skimming files.

A preposed solution to fulfill that goal would be:  
```javascript
'use strict';

module.exports = 'c2fo.ui.modules.pager';

angular.module(module.exports, [])
    .directive('...', function(){...})
    .controller('...', function(){...});
```